src/public/styles.css

```diff
@@ -9,21 +11,41 @@ body {
   color: var(--clr-primary);
   font-family: Poppins, sans-serif;
   font-size: 1rem;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: var(--clr-primary);
 }
 
+/* todo.html styling */
```

Comment:
nice to see comments like this, makes life easier for other developers

Reply:
It's the only way I can see where I've put everything!